### DISCUSSION TITLE
takara-mouse-rna-bcr-smarter

### DISCUSSION THREAD

**Discussion by pbpayal** [milaboratory/mixcr#1589] [Q&A] [answered]
## Checklist before submitting the issue:

 I am running mixcr/4.3.2. I want to run the "takara-mouse-rna-bcr-smarter" preset because the library was prepared using **SMARTer Mouse BCR IgG H/K/L Profiling Kit**. But I am getting error.

## ERROR
It says this preset doesn't exist.
*IMPORTANT: MiXCR will use at most 12000MB of RAM,
           use -Xmx to change automatically set heap size (i.e. -Xmx50g to set heap size to 50 Gb)
        No preset with name "takara-mouse-rna-bcr-smarter".
        Here are supported presets with similar names:
        - takara-mouse-bcr-cdr3
- takara-mouse-tcr-cdr3
- takara-mouse-bcr-full-length*

## Exact MiXCR commands 

*mixcr analyze takara-mouse-rna-bcr-smarter \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result *

What preset am I supposed to use for this kit then?

**Comment by mizraelson** [comment, marked as answer]
Hi,
Please update to the latest MiXCR V4.6. The command should work with it.

**Comment by pbpayal** [comment]
Problem is this package is on a server not managed by me. Let me see.

**Comment by mizraelson** [comment]
You can use `takara-mouse-bcr-full-length` in that case, but I would strongly recommend upgrading.
